### PR TITLE
Portfolio Management (Extended filters & Archive page changes)

### PR DESCRIPTION
**Pull request by hikmet-erd**
Changes related to extended filters and excel + archive page changes

### CODE REVIEW

File: content/en/docs/developerportal/portfolio-management/_index.md [hunk 1]
```diff
@@ -459,6 +459,8 @@ The list contains the following information:
 * **Archiving Reason** – This shows reason why the initiative was archived.
 * **Archived Date** – This shows the date when the initiative was archived.
 * **Archived By** – This shows the avatar of the user who archived the initiative.
+* Icon of the linked apps – If there is only one linked app, clicking the icon shows the information about the app. If there are multiple apps or no linked apps, the icon is not clickable.
```
Comment (riosjuan) [inline content/en/docs/developerportal/portfolio-management/_index.md:462]: Text should be in bold `**Icon of the the linked apps**`

File: content/en/docs/developerportal/portfolio-management/_index.md [hunk 2]
```diff
@@ -459,6 +459,8 @@ The list contains the following information:
...
+* Avatar of the initiative owner – Hovering over the avatar shows the name of the initiative owner.
```
Comment (riosjuan) [inline content/en/docs/developerportal/portfolio-management/_index.md:463]: Text should be in bold `**Avatar of the initiative owner**`

File: content/en/docs/developerportal/portfolio-management/export-import-initiatives.md [hunk 3]
```diff
@@ -70,30 +70,31 @@ All the details of an initiative are exported, except the data of the following
...
+   | **WSJFTimeCriticality** | {{< icon name="remove-circle-filled" color="red" >}} | String                       | The selectable options can be found in the [WSJF](/developerportal/portfolio-management/prioritization-models/#wsjf) section in *Prioritization Models*. |                               |`Highest`|
+   | **WSJFRiskReduction** | {{< icon name="remove-circle-filled" color="red" >}} | String                       | The selectable options can be found in the [WSJF](/developerportal/portfolio-management/prioritization-models/#wsjf) section in *Prioritization Models*. |                                |`Medium`|
+   | **WSJFJobSize**     | {{< icon name="remove-circle-filled" color="red" >}} | String                       | The selectable options can be found in the [WSJF](/developerportal/portfolio-management/prioritization-models/#wsjf) section in *Prioritization Models*. |                                    |`XS`|
+
+6. When your Excel file is ready to be imported, drag it to the **Import Initiatives** dialog box, or click **Upload** and select the file.
```
Comment (riosjuan) [inline content/en/docs/developerportal/portfolio-management/export-import-initiatives.md:96]: `6. When your Excel file is ready to be imported, drag it to the **Import Initiatives** dialog box, or click **Upload** and select the file you want to import.`

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 4]
```diff
@@ -17,6 +17,15 @@ To see the current status of the Mendix Developer Portal, see [Mendix Status](ht
 
 ## 2024
 
+### March 28
+
+#### Features
+
+* We made the following improvements to [Portfolio Management](/developerportal/portfolio-management/):
+  * On the **Archive** page, you can now export and filter the archived initiatives.
```
Comment (riosjuan) [inline content/en/docs/releasenotes/developer-portal/_index.md]: `* On the **Archive** page, you can now filter and export the archived initiatives.`

**Review by Luyao-Zhang-1 — APPROVED** [review]

**Comment by MarkvanMents** [comment]
Released.

**Pull request merged**